samples_project/Assets/SampleViewer/Samples/FlightSim/readme.md

```diff
@@ -0,0 +1,29 @@
+# Third Person Character Controller
+
+Allows users to explore a city/map from the perspective of a Third Person Character.
+
+![Image of Flight Simulator](FlightSim.jpg)
+
+## How it works
+
+1. Check the box for **Mesh Colliers Enabled** on the `ArcGIS Map` component.
+2. Pick an airport to take off from and set the latitude and longitude of the airport as your world center
```

Comment:
Possibly add a link to this site: https://www.world-airport-codes.com/
@mrnelson6 is that ok?